docs/install/install.md

```diff
@@ -234,6 +234,19 @@ Go and explore your InvenioRDM instance at [https://127.0.0.1:5000](https://127.
     By default, the host is `127.0.0.1` and the port is `5000`. Pass `--host` and `--port`
     to change them e.g., `invenio-cli run --host 127.0.0.2 --port 5001`.
 
+### Admin account
+
```

Comment:
FYI `invenio-cli services setup` actually automatically creates an admin user: 
- https://github.com/inveniosoftware/invenio-cli/blob/master/invenio_cli/commands/services.py#L153  -->
- https://github.com/inveniosoftware/invenio-rdm-records/blob/master/invenio_rdm_records/fixtures/users.py#L22 

and there's a whole issue about it: https://github.com/inveniosoftware/invenio-rdm-records/issues/425

Highlighting the CLI commands like was done is good though.